src/main/java/rs/hybridit/model/Book.java

```diff
@@ -0,0 +1,116 @@
+package rs.hybridit.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(catalog = "dbhybridlibrary", name = "books")
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "isbn")
```

Comment:
It would be good to include some field validation and constraints (fileds which are not empty for example)